cmd/ipfs/daemon.go

```diff
@@ -457,6 +469,99 @@ func daemonFunc(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment
 	return errs
 }
 
+const mfsRepinInterval = time.Minute
+
+func pinMFSOnChange(cctx *oldcmds.Context, node *core.IpfsNode) (<-chan error, error) {
+	errCh := make(chan error)
+	go func() {
+		var lastRootCid cid.Cid
```

Comment:
Do you think it might be good to `defer close(errCh)` here, so that a routine waiting on `errCh` can know that all errors were collected, upon seeing the channel closed?

Reply:
Absolutely. Thanks.